packages/cli/src/utils/exchange.ts

```diff
@@ -24,18 +25,19 @@ export async function checkNotDangerousExchange(
   amount: BigNumber,
   largeOrderPercentage: number,
   deppegedPricePercentage: number,
-  buyBucket: boolean
+  buyBucket: boolean,
+  stableTokenInfo: StableTokenInfo = stableTokenInfos[StableToken.cUSD]
 ): Promise<boolean> {
   const oracles = await kit.contracts.getSortedOracles()
-  const exchange = await kit.contracts.getExchange()
-  const oracleMedianRate = (await oracles.medianRate(CeloContract.StableToken)).rate
+  const exchange = await kit.contracts.getExchange(stableTokenInfo.symbol as StableToken)
```

Comment:
I think this is preferable as an API. `getContract` can't return a typed instance, you have to manually cast.

We also do `await kit.contracts.getGoldToken` for example, rather than `(await kit.contracts.getContract(GoldToken)) as GoldTokenWrapper`

Reply:
ahh makes sense!